cmd/execute-cadence-script/README.md

```diff
@@ -0,0 +1,51 @@
+# Execute Cadence Script
+
+## Description
+
+This utility binary can be used to execute a Cadence script against a running Flow network. By default, access nodes are expected on localhost:3569. Script can be executed on an arbitrary block height.
+
+## Usage
+
+```sh
+Usage of ./execute-cadence-script:
+  -a, --api string           access node API address (default "localhost:3569")
```

Comment:
I prefer using `127.0.0.1` here. I don't remember why, but years ago, I learned that sometimes it avoids some problems.